scheduler-server/sql/scheduler/tables/scheduling_rule.sql

```diff
@@ -0,0 +1,51 @@
+create table scheduling_rule (
```

Comment:
:large_orange_diamond: Random question: is there a reason the path to these files is `scheduler-server/sql/scheduler/tables/*`, rather than just `scheduler-server/sql/tables/*` ? Does the addition of `scheduler` provide some delineation I'm not aware of?

Reply:
Good question! Yes, the `scheduler` directory is needed to maintain parity with the directory structure that `deployment/postgres-init-db/sql/<table_name>` needs. Since `<table_name>` may differ from the Gradle subproject name (`merlin-server` vs. `merlin` for example) this extra layer is needed :)

Reply:
Could rework this to just be `scheduler-server/sql/tables/*` and then have `scheduler-server/build.gradle` define a DB table name variable, but this feels a little more natural than relying on the build system for that.